bundles/org.openhab.binding.emby/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,156 @@
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="controller"/>
+		</supported-bridge-type-refs>
+		<label>EMBY Device</label>
+		<description>This is a player device which connects to an EMBY server.</description>
+		<semantic-equipment-tag>MediaPlayer</semantic-equipment-tag>
+		<channels>
+			<channel id="control" typeId="control"/>
+			<channel id="stop" typeId="stop"/>
+			<channel id="title" typeId="title"/>
+			<channel id="mute" typeId="mute"/>
+			<channel id="show-title" typeId="show-title"/>
+			<channel id="image-url" typeId="image-url"/>
+			<channel id="current-time" typeId="current-time"/>
+			<channel id="duration" typeId="duration"/>
+			<channel id="media-type" typeId="media-type"/>
+		</channels>
+		<config-description>
+			<parameter name="deviceID" type="text" required="true">
+				<label>DeviceID</label>
+				<description>This is the deviceId you want to connect to.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>Title of the current song</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="show-title">
+		<item-type>String</item-type>
+		<label>Show Title</label>
+		<description>Title of the current show</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="control">
+		<item-type>Player</item-type>
+		<label>Control</label>
+		<description>Control the Emby Player, e.g. start/stop/next/previous/ffward/rewind</description>
+		<category>Player</category>
+	</channel-type>
+
+	<channel-type id="stop">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops the player. ON if the player is stopped.</description>
+	</channel-type>
+
+	<channel-type id="mute">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Mute/unmute your device</description>
+	</channel-type>
+
+	<channel-type id="current-time">
+		<item-type>Number:Time</item-type>
+		<label>Current Time</label>
+		<description>Current time of currently playing media</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="duration">
+		<item-type>Number:Time</item-type>
+		<label>Duration</label>
+		<description>Length of currently playing media</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="media-type">
+		<item-type>String</item-type>
+		<label>Media Type</label>
+		<description>Media type of the current file</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="image-url">
+		<item-type>String</item-type>
+		<label>image url</label>
+		<description>The url of the playing media</description>
+		<state readOnly="true" pattern="%s"/>
+		<config-description>
+			<parameter name="image-url_type" type="text">
```

Comment:
These parameters need some work, this comment is just for reference to the other comment in the binding constants.